client/extensions/woocommerce/state/ui/shipping/zones/test/selectors.js

```diff
@@ -41,12 +41,42 @@ describe( 'selectors', () => {
 			const state = {
```

Comment:
It's not about a multi-site UI, but about viewing the stores for separate sites. They'll both end up in the state, under 2 different site IDs.